ccx_messaging/consumers/kafka_consumer.py

```diff
@@ -45,11 +43,19 @@ def __init__(
         )
 
         # Confluent initialization
-        config = {
-            "bootstrap.servers": kwargs.get("bootstrap_servers", ""),
-            "group.id": kwargs.get("group_id", None),
-            "retry.backoff.ms": retry_backoff_ms,
-        }
+        
+        config = kwargs.copy()
```

Comment:
why is copying from kwargs needed instead of just using the 3 args we need?

Reply:
We don't know the amount of arguments that we will need in advance. Clowder configuration will inject, at least:

Reply:
Given that, I'm trying to think in a long-term fix for the Clowder configuration mess that allow us to decouple the clowder configuration and the consumer and producers configuration

Reply:
are all the clowder arguments compatible with confluent consumer/producer?